climakitaegui/core/data_interface.py

```diff
@@ -233,12 +236,15 @@ def scenario_view(self):
 
         else:
             # Set time range of historical data
-            if self.downscaling_method == "Statistical":
-                historical_climate_range = self.historical_climate_range_loca
-            elif self.downscaling_method == "Dynamical+Statistical":
-                historical_climate_range = self.historical_climate_range_wrf_and_loca
-            else:
-                historical_climate_range = self.historical_climate_range_wrf
+            match self.downscaling_method:
+                case "Dynamical":
+                    historical_climate_range = self.historical_climate_range_wrf
+                case "Statistical":
+                    historical_climate_range = self.historical_climate_range_loca
+                case "Dynamical+Statistical":
```

Comment:
should replace 
"Dynamical+Statistical"
with `_`
```python
match self.downscaling_method:
    case "Dynamical":
        # stuff
    case "Statistical":
        # stuff
    case _:
        # assume "Dynamical+Statistical" unless we need to do param validation here
```

Reply:
There are only 3 possible choices. I can add a `case _`

Reply:
Added `case _`. I think it is more clear with each choice spelled out but happy to defer to other approach...